### PR TITLE
AH: add some missing actions for datarepo

### PR DESCRIPTION
**Pull request by ahaessly**
no ticket
add some missing actions and an admin role
---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

**Comment by dvoet** [comment]
what's the difference between `delete` and `delete_dataset` or `delete_study`

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -408,32 +408,50 @@ resourceTypes = {
...
+      }
       "alter_policies" = {
         description = "Can alter policies"
       }
+      "delete" = {
+        description = "Can delete policies/resources"
+      }
     }
-    ownerRoleName = "steward"
+    ownerRoleName = "admin"
```
Comment (gpolumbo-broad) [inline src/main/resources/reference.conf:433]: Is this going to have an impact on existing resources?  Will there need to be a migration to change the role of existing owners?

Comment (ahaessly) [reply src/main/resources/reference.conf:433]: we aren't in production yet. so no.

**Review by gpolumbo-broad — APPROVED** [review]

**Review by thathert — APPROVED** [review]
LGTM

**Pull request merged**